project.godot

```diff
@@ -92,8 +92,8 @@ config/icon="res://icon.png"
 
 [autoload]
 
-GameState="*res://state/GameState.gd"
 Constants="*res://lib/Constants.gd"
+GameState="*res://state/GameState.gd"
```

Comment:
Load Constants before GameState so that Constants can be used when initializing GameState